pkg/api/persistentvolumeclaim/util_test.go

```diff
@@ -244,50 +218,15 @@ func TestAnyDataSourceFilter(t *testing.T) {
 			anyEnabled: true,
 			want:       genericDataSource,
 		},
-		"snapshot enabled with snapshot ds": {
-			spec:            core.PersistentVolumeClaimSpec{DataSource: snapshotDataSource},
-			snapshotEnabled: true,
-			want:            snapshotDataSource,
-		},
-		"snapshot enabled with generic ds": {
-			spec:            core.PersistentVolumeClaimSpec{DataSource: genericDataSource},
-			snapshotEnabled: true,
-			want:            nil,
-		},
-		"both enabled with empty ds": {
```

Comment:
should this be changed to "any enabled with empty ds"?

Reply:
This test is removed though.  ?